src/components/error/error.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ErrorType } from "../app";
+import "./error.scss";
+
+interface IProps {
+  type: ErrorType
+}
+
+export const errorMsg: Record<ErrorType, string> = {
+  auth: "We can't establish a connection to your account.",
+  network: "Your network connection has been lost or interrupted.",
+  timeout: "Your session has expired."
+};
+
+export const Error: React.FC<IProps> = ({ type }) => {
+  return (
+    <div className="error" data-cy="error">
+      <h1>Hmm... we&apos;re having trouble connecting.</h1>
```

Comment:
I would probably just say "We're having trouble connecting." but don't feel strongly about it.

Reply:
I have no opinion here, but this exact wording comes from Kiley who designed this error message:
https://www.pivotaltracker.com/story/show/174052604/comments/218220039
and it's also used by Micheal in UI mocks. So, I think I'll leave it for now. Easy to change later too.